web/rainmaker/dev-packages/egov-estate-dev/src/ui-config/screens/specs/estate-citizen/EstateBranch_OwnershipTransfer_FamilySettlement.json

```diff
@@ -3,6 +3,63 @@
   "moduleName": "EstatePropertyService",
   "EstateBranch_OwnershipTransfer_FamilySettlement": [
     {
+      "dataSources": [
+        {
+          "name": "typesOfOwner",
+          "type": "local",
+          "cacheable": true,
+          "lazyload": true,
+          "values": [
+            {
+              "code": "New"
+            },
+            {
+              "code": "Existing"
+            }
+          ]
+        },
+        {
+          "name": "modeOfTransfer",
+          "type": "local",
+          "cacheable": true,
+          "lazyload": true,
+          "values": [
+            {
+              "code": "FAMILY_SETTLEMENT"
+            },
+            {
+              "code": "COURT_DECREE"
+            }
+          ]
+        },
+        {
+          "name": "owners",
+          "type": "path",
+          "jsonPath": "propertyDetails.owners",
+          "code": "ownerDetails.ownerId",
+          "label": "ownerDetails.ownerName"
+        },
+        {
+          "name": "legalHeirRelationWithDeceased",
```

Comment:
No need of legalHeirRelationwithDeceased and benificiaryRelationwithDeceased datasources for this application

Reply:
ok. i change and recommit